ursula_cli/shell.py

```diff
@@ -260,6 +261,10 @@ def _run_heat(args, hot):
         'template': hot
     }
 
+    if args.heat_parameters:
+        print args.heat_parameters
```

Comment:
Is this a leftover debug print statement?  If you want to print this out for the user, the app tends to use (sorta consistently), the `LOG` global which is a `logging.Logger` object.  

If that's what you want to use, `LOG.debug()` is probably the right call for you.